.env.sample

```diff
@@ -177,3 +177,6 @@ RATE_LIMITER_ENABLED=true
 # Configure default throttling parameters for rate limiter
 RATE_LIMITER_REQUESTS=1000
 RATE_LIMITER_DURATION_WINDOW=60
+
+# Feature flag to enable new publish UI
+PUBLISH_UI_ENABLED=false
```

Comment:
Temporarily added. Acts as a feature flag. Can get rid off when we have some basic feature flag support.